sci-mathematics/cgal/cgal-4.10.2.recipe

```diff
@@ -20,32 +20,36 @@ COPYRIGHT="2005 Stanford University (USA)
 	1997-2004 ETH Zurich (Switzerland)
 	1997-2004 Tel-Aviv University (Israel)"
 LICENSE="GNU GPL v3
-	GNU LGPL v3"
+	GNU LGPL v3
+	Boost
+	MIT"
 REVISION="1"
-SOURCE_URI="https://github.com/CGAL/cgal/releases/download/releases%2FCGAL-$portVersion/CGAL-$portVersion.tar.xz"
+SOURCE_URI="https://github.com/CGAL/cgal/releases/download/releases/CGAL-$portVersion/CGAL-$portVersion.tar.xz"
 CHECKSUM_SHA256="5cc666e9813cb6515c1b4d14a167160be24d60c43c97fef8970dd3798a4c66a8"
 SOURCE_DIR="CGAL-$portVersion"
 
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="?x86"
 
-if [ "$targetArchitecture" != x86_gcc2 ]; then
-	commandSuffix=$secondaryArchSuffix
-	commandBinDir=$binDir
-else
-	commandSuffix=
-	commandBinDir=$prefix/bin
+libVersion="13.0.0"
```

Comment:
According to the [HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering) :
```diff
-libVersion="13.0.0" 
-libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
-
 commandSuffix=$secondaryArchSuffix
 commandBinDir=$binDir
 if [ "$targetArchitecture" = x86_gcc2 ]; then
  	commandSuffix=
  	commandBinDir=$prefix/bin
 fi
+
+libVersion="13.0.0" 
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
```